kv/dist_sender_test.go

```diff
@@ -203,8 +203,12 @@ func TestSendRPCOrder(t *testing.T) {
 
 	var testFn rpcSendFn = func(opts rpc.Options, method string,
 		addrs []net.Addr, _ func(addr net.Addr) interface{},
-		_ func() interface{}, _ *rpc.Context) ([]interface{}, error) {
-		return nil, verifyCall(opts, addrs)
+		getReply func() interface{}, _ *rpc.Context) ([]interface{}, error) {
+		err := verifyCall(opts, addrs)
+		if err == nil {
```

Comment:
Usually this would be structured as:

```
  if err := verifyCall(opts, addrs); err != nil {
    return nil, err
  }
  return []interface{}{getReply()}, nil
```